### ISSUE TITLE
usage in concurrent application

### ISSUE THREAD

**Issue by alext234** [gojek/heimdall#48]
Hi guys, I don't see any example of concurrent usage.  Is it safe to use the same the same client object from multiple go-routine?

**Comment by dcu** [comment]
yes it's safe and actually that's how you do it otherwise you'd not be using the pool of connections.
